test.sh

```diff
@@ -0,0 +1,12 @@
+export OFI_INTERFACE=eth0
```

Comment:
(lint) Tips depend on target shell and yours is unknown. Add a shebang. [SC2148]